dev/src/dev/portal.clj

```diff
@@ -1,6 +1,8 @@
 (ns dev.portal
   (:require [portal.api :as p]))
 
+(set! *warn-on-reflection* true)
```

Comment:
Why was this required? I don't see any interop

Reply:
It's [not in the diff](https://github.com/metabase/metabase/pull/50470/files/53802612c8f01e94a303d7521e97419b617e760b#diff-022365b6dcb53c61bbe8524b40fed905a99f833b37b836c22cdc0457726613ebR72), I just angered the linter gods.